CUSEC_Twitter/hashtags.txt

```diff
@@ -1,2 +1,2 @@
-#CUSEC2015
-#CUSEC2015
+#cats
+#dogs
```

Comment:
I would prefer to leave this as #cusec2015, just because the description says it was for the conference